ingestion/src/metadata/ingestion/source/database/dbt/dbt_config.py

```diff
@@ -173,7 +173,7 @@ def _(config: DbtCloudConfig):  # pylint: disable=too-many-locals
             params_data["job_definition_id"] = job_id
 
         response = client.get(f"/accounts/{account_id}/runs", data=params_data)
-        if not response and not response.get("data"):
+        if not response or not response.get("data"):
```

Comment:
This would throw an error if there is `response` is `None`

Reply:
Yes it should throw an error if response is `None`

Reply:
NoneType won't have get, some thing like that, correct me if wrong